edx_lint/files/common_constraints.txt

```diff
@@ -7,6 +7,10 @@
 # idea to pin this package across all edx repos, Ideally, link to other information
 # that will help people in the future to remove the pin when possible.  
 # Writing an issue against the offending project and linking to it here is good.
+#
+# Note: Changes to this file will automatically be used by other repos, referencing
+#  this file fromt Github directly. It does not require packaging in edx-lint.
```

Comment:
typo

Reply:
Fixed "fromt", assuming that was the typo.